user-manual/transfer/import-metadata.rst

```diff
@@ -263,6 +264,199 @@ metadata. The JSON file is added to the metadata directory as above.
+     is used during metadata reingests to update or delete existing metadata.
+
+   This is the ``source-metadata.csv`` file we are going to use for the
+   ``metadataTransfer`` directory from above.
+
+   .. csv-table::
+      :file: _csv/source-metadata.csv
+      :header-rows: 1
+
+#. Create the two metadata XML files listed in the ``metadata`` column of the
+   ``source-metadata.csv`` file: ``beihai.xml`` and ``bird-dc.xml``.
+
+   The ``beihai.xml`` metadata file contains unstructured text represented
+   as a single top level ``<metadata>`` element with no XML namespace wrapping
+   `character data <https://www.w3.org/TR/REC-xml/#sec-cdata-sect>`_ like
+   this:
+
+   .. literalinclude:: scripts/metadata-xml-unstructured.xml
+      :language: xml
+
+   And the ``bird-dc.xml`` metadata file contains
+   `OAI-PMH <https://www.openarchives.org/pmh/>`_ Dublin Core metadata that
+   could be validated against the
+   `OAI-PMH Version 2.0 specification <https://www.openarchives.org/OAI/openarchivesprotocol.html>`_
+   and it looks like this:
+
+   .. literalinclude:: scripts/metadata-xml-dc.xml
+      :language: xml
+
+#. Create a subdirectory called ``metadata`` and place the ``source-metadata.csv``
+   file and the two metadata XML files inside.::
+
+    metadataTransfer/
+    ├── audio
+    │   └── bird.mp3
+    ├── beihai.tif
+    └── metadata
+        ├── source-metadata.csv
+        ├── beihai.xml
+        └── bird-dc.xml
+
+#. :ref:`Process the transfer <process-transfer>` as usual.
+
+The METS file for the above example includes two descriptive metadata sections
+(``<dmdSec>``), one for each file in the transfer. Both sections are given
+attributes that specify their creation time and their status is set as
+``original``. And the contents of the metadata XML files in the ``metadata``
+directory have been transposed here.
+
+.. literalinclude:: scripts/metadata-xml-dmdsecs.xml
+   :language: xml
+
+Note in the metadata wrapper (``<mdWrap>``) that they are given an MDTYPE of
+"OTHER" and an MDOTHERTYPE with the value of the ``type`` cell from the
+``source-metadata.csv`` file. This is useful for updating or deleting the
+metadata files in the AIP during metadata reingests.
+
+Updating metadata through metadata reingests
+++++++++++++++++++++++++++++++++++++++++++++
+
+You can update the metadata contained in the METS file by reingesting your
+AIP and adding a new ``source-metadata.csv`` file and updated metadata XML
+files from one of your transfer source locations before approving the reingest.
+
+#. This is the new ``source-metadata.csv`` file we are going to use to
+   update the metadata associated with the ``beihai.tif`` digital object
+   in the previous transfer.
+
+   .. csv-table::
+      :file: _csv/source-metadata-update.csv
+      :header-rows: 1
+
+   It's important that in the ``type`` column we reuse the identifier
+   (i.e. ``beihai``) that was set as the  ``OTHERMDTYPE`` attribute of the
```

Comment:
Just make sure to change beihai as the type if you change it as I suggested in the source-metadata.csv